docs/network/interfaces_and_networks.md

```diff
@@ -632,161 +544,18 @@ has no DHCP server at all. Therefore, the VM won't have the search domains infor
-queue private to a specific vCPU.
-
-Without enabling the feature, network performance does not scale as the
-number of vCPUs increases. Guests cannot transmit or retrieve packets in
-parallel, as virtio-net has only one TX and RX queue.
-
-Virtio interfaces advertise on their status.interfaces.interface entry a field named queueCount.  
-The queueCount field indicates how many queues were assigned to the interface.  
-Queue count value is derived from the domain XML.  
-In case the number of queues can't be determined (i.e interface that is reported by quest-agent only),
-it will be omitted.
-
-
-*NOTE*: Although the virtio-net multiqueue feature provides a
-performance benefit, it has some limitations and therefore should not be
-unconditionally enabled
-
-#### Some known limitations
-
--   Guest OS is limited to ~200 MSI vectors. Each NIC queue requires a
-    MSI vector, as well as any virtio device or assigned PCI device.
-    Defining an instance with multiple virtio NICs and vCPUs might lead
-    to a possibility of hitting the guest MSI limit.
-
--   virtio-net multiqueue works well for incoming traffic, but can
-    occasionally cause a performance degradation, for outgoing traffic.
-    Specifically, this may occur when sending packets under 1,500 bytes
-    over the Transmission Control Protocol (TCP) stream.
-
--   Enabling virtio-net multiqueue increases the total network
-    throughput, but in parallel it also increases the CPU consumption.
-
--   Enabling virtio-net multiqueue in the host QEMU config, does not
-    enable the functionality in the guest OS. The guest OS administrator
-    needs to manually turn it on for each guest NIC that requires this
-    feature, using ethtool.
-
--   MSI vectors would still be consumed (wasted), if multiqueue was
-    enabled in the host, but has not been enabled in the guest OS by the
-    administrator.
-
--   In case the number of vNICs in a guest instance is proportional to
-    the number of vCPUs, enabling the multiqueue feature is less
-    important.
-
--   Each virtio-net queue consumes 64 KiB of kernel memory for the vhost
-    driver.
-
-*NOTE*: Virtio-net multiqueue should be enabled in the guest OS
-manually, using ethtool. For example:
-`ethtool -L <NIC> combined #num_of_queues`
-
-More information please refer to [KVM/QEMU
-MultiQueue](http://www.linux-kvm.org/page/Multiqueue).
-
 ### sriov
 
-In `sriov` mode, virtual machines are directly exposed to an SR-IOV PCI
-device, usually allocated by [Intel SR-IOV device
-plugin](https://github.com/intel/sriov-network-device-plugin). The
-device is passed through into the guest operating system as a host
-device, using the
+In `sriov` mode, SR-IOV Virtual Functions' PCI devices are directly exposed to virtual machines.
+[SR-IOV device
+plugin and CNI](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin) can be used to manage SR-IOV devices in kubernetes, making them available for kubevirt to consume.
```

Comment:
The text says device-plugin and CNI but it refers to the device-plugin repo, could you please fix its so there is a link for each component?